src/main/scala/no/ndla/articleimport/service/converters/HTMLCleaner.scala

```diff
@@ -436,6 +437,15 @@ trait HTMLCleaner {
       }
     }
 
+    private def unwrapNestedPs(element: Element): Unit = {
```

Comment:
Mulig dette ikke gjør noe, men vil ikke dette endre på artikler som har flere p-tags i en ptag for avsnitt? 
`<p><p>Hei</p><p>Hei</p></p>` blir vist som:
```
Hei
Hei
```

Vil nå bli `<p>Hei Hei</p>` som blir vist:
```
Hei Hei
```